### ISSUE TITLE
Datatable sur la liste des fiches taches - installation et tableau simple

### ISSUE THREAD

**Issue by remipichon** [assomaker/manifmaker#13]
~~installer Datatable pour Meteor~~
https://atmospherejs.com/aslagle/reactive-table

mettre en place un datatable pour la liste des tasks qui affiche les informations suivantes dans l'ordre : 
- ~~name~~
- group
- ~~team~~
- ~~nombre de creneaux existant~~

Les colonnes suivantes sont triables alphabétiquement : 
- ~~name~~
- task group
- ~~team~~

la doc : 
https://github.com/aslagle/reactive-table/

**Comment by ghost** [comment]
je croyais qu'on s'occupait pas du task group encore. 

Il faut que je crée une collection? (une collection de tasks?)

**Comment by remipichon** [comment]
Fais donc une collection de pour les groupes de tâches.

On fait les groupes oui, ce sont les compétences qu'on ne fait pas. J'ai
pas mis les groupes de tâches dans le cahier des charges.

Dans la mesure où les groupes sont pas fait, ne les prends pas en compte
dans la issue datatable

**Comment by remipichon** [comment]
- affiicher le nom de l'equipe au lieu de l'ID
- reduire la taille de la colonne nb de creneau
- mettre le colonne groupe en premier
- remplacer le "show 10 rows per pages" pour afficher la dropdown avec les choix de bases (10, 20, 50, 100)
- resoudre le CSS "page 1 of 1" (ca le nombre s'affiche mal)
- effectuer la recherche uniquement sur le champ name

**Comment by cpalliere** [comment]
J'imagine que pour afficher le nom de l'équipe, tu veux que je fasse ça sans changer le constructeur de Task. Ou j'ai le droit?

**Comment by remipichon** [comment]
Nope, il faut laisser le constructeur comme ca car l'objet que le
constructeur crée est celui que nous stockons en BDD. Nous devons stocker
un _id de team pour les tasks car si le label de la team change il faut
pouvoir répercuter ca sur toutes les tasks avec le moins de modifications.
En stockant l'_id on va chercher directement le dernier label à jour en
BDD.

## 

Remi Pichon
